src/foam/swift/SWIFTOutputterImpl.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * NANOPAY CONFIDENTIAL
+ *
+ * [2021] nanopay Corporation
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of nanopay Corporation.
+ * The intellectual and technical concepts contained
+ * herein are proprietary to nanopay Corporation
+ * and may be covered by Canadian and Foreign Patents, patents
+ * in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from nanopay Corporation.
+ */
+
+foam.CLASS({
+    package: 'foam.swift',
+    name: 'SWIFTOutputterImpl',
+
+    implements: [
+        'foam.swift.SWIFTOutputter'
+    ],
+
+    javaImports: [
+        'foam.core.*',
+        'java.util.List',
+        'java.util.Date',
+    ],
+
+    properties: [
+        {
+            class: 'Class',
+            name: 'of',
+            required: true
+        },
+    ],
+
+    methods: [
+        {
+            name: 'outputValue',
+            code: function(message) {
+                var SBF = '';
```

Comment:
and now as we have generateBasicHeader() and generateApplicationHeaderInput()(hopefully)  can you suggest how to make our output to look exactly like example in Parsing a [SWIFT Message document](https://www.immagic.com/eLibrary/ARCHIVES/GENERAL/IONA_IE/I070510D.pdf) page 2?